iquip/apps/scheduler.py

```diff
@@ -0,0 +1,352 @@
+        return mime
+
+    def dropMimeData(self,  # pylint: disable=too-many-arguments
+        mimedata: QMimeData,
+        action: QAction,
+        row: int,
+        column: int,  # pylint: disable=unused-argument
+        parentIndex: QModelIndex  # pylint: disable=unused-argument
+    ) -> Literal[True]:
+        """Overridden.
+
+        Changes the priority of the experiments.
+        
+        Args:
+            mimedata: The QMimeData instance containing str value of the pre-selected index.
+            action: The QtAction instance classifying the action 
+              (for terminating the function when it is not dropped in the appropriate region)
+            row: The target row that is to be changed with the experiment in mimedata.
+            column: Not used.
+            parentIndex: Not used.
+
+        Returns:
+            True value.
+        """
+        idx = int(mimedata.text())
+        if action == Qt.IgnoreAction:
+            return True
+        if row < 0:
+            row = self.rowCount()
+        if (self.experimentData[idx].priority()
+            != self.experimentData[min(row, self.rowCount() - 1)].priority()):
+            return True
+        if idx != row:
+            item = self.experimentData.pop(idx)
+            self.experimentData.insert((row - 1) if idx < row else row, item)
+
+        # TODO(giwon2004): emit signal for change of priority through artiq-proxy.
+        return True
+
+    def flags(self, index: QModelIndex) -> int:
+        """Overridden."""
+        defaultFlags = Qt.ItemIsSelectable | Qt.ItemIsEnabled
+        if index.isValid():
+            return Qt.ItemIsEditable | Qt.ItemIsDragEnabled | defaultFlags
+        return Qt.ItemIsDropEnabled | defaultFlags
+
+
+class ExperimentDelegate(QAbstractItemDelegate):
+    """Delegate for displaying the layout of each data in the experiment list.
+
+    TODO(giwon2004): Enabling buttons when displayed using QAbstractDelegate.
+    """
+
+    def paint(self,
+        painter: QPainter,
+        option: QStyleOptionViewItem,
+        index: QModelIndex):
+        """Overridden."""
+        expView = index.data(Qt.DisplayRole)
+        expView.resize(option.rect.size())
+        painter.save()
+        painter.translate(option.rect.topLeft())
+        expView.render(painter)
+        painter.restore()
+
+    def sizeHint(self, option: QStyleOptionViewItem, index: QModelIndex) -> QSize:  # pylint: disable=unused-argument
+        """Overridden."""
+        expView = index.data(Qt.DisplayRole)
+        return expView.sizeHint()
+
+
+class SchedulerApp(qiwis.BaseApp):
+    """App for displaying the submitted experiment queue.
+
+    Attributes:
+        schedulerFrame: The frame that shows the submitted experiment queue.
+    """
+
+    def __init__(self, name: str, parent: Optional[QObject] = None):
+        """Extended.
+
+        Args:
+            name: The name of the app.
+            parent: The QObject that contains this app."""
```

Comment:
You __must not__ write the arguments docstring of its super class, i.e., `name` and `parent`.

It's susceptible for changing the docstring of `BaseApp` because once the docstring of `BaseApp` is updated, we should update all docstring of its subclasses manually.

Thus, we decided to use the `Extended` expression and unexplained arguments imply that they are in the super class.